src/sdk/pynni/nni/nas/benchmarks/nds/model.py

```diff
@@ -52,8 +53,8 @@ class NdsTrialConfig(Model):
         'residual_basic',
         'vanilla',
     ])
-    model_spec = JSONField(index=True)
-    cell_spec = JSONField(index=True, null=True)
+    model_spec = JSONField(json_dumps=json_dumps, index=True)
```

Comment:
why this modification can resolve the dict element sequence issue?

Reply:
The comparison of JSON field is based on string comparison, and to compare two dumped JSON of unordered dict, they need to be sorted ahead.